src/main/g8/build.sbt

```diff
@@ -15,6 +15,8 @@ lazy val root = (project in file(".")).
       "com.typesafe.akka" %% "akka-stream"          % akkaVersion,
 
       "com.typesafe.akka" %% "akka-http-testkit" % akkaHttpVersion % Test,
+      "com.typesafe.akka" %% "akka-testkit" % akkaVersion % Test,
+      "com.typesafe.akka" %% "akka-stream-testkit" % akkaVersion % Test,
```

Comment:
Could you align the `%` characters?

Reply:
Done, but it is not reflected in the pull request. Do you know why?

> Add more commits by pushing to the 10.0.x branch on angelcervera/akka-http-quickstart-scala.g8.

Reply:
https://github.com/akka/akka-http-quickstart-scala.g8/pull/29

Reply:
Hmm, not sure, perhaps creating a branch before creating the PR might have helped? Thanks for creating the separate PR